client/my-sites/stats/annual-highlights-section/index.tsx

```diff
@@ -58,6 +71,21 @@ export default function AnnualHighlightsSection( { siteId }: { siteId: number }
 	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) );
 	const viewMoreHref = siteSlug ? `/stats/annualstats/${ siteSlug }` : null;
 
+	const previousYear = insights?.years?.find( ( y ) => y.year === ( year - 1 ).toString() );
+	const hidePreviousArrow = ! previousYear;
+	const currentYear = new Date().getFullYear();
```

Comment:
How does this differ from the `year` const? 🤔

Reply:
`year` only defaults to the current year if no date is passed to the component. `currentYear` is used to check if an arrow should be active or disabled.

Reply:
This has changed in the [new approach](https://github.com/Automattic/wp-calypso/pull/71343), but I used a similar way with one variable for testing if an arrow should be disabled and one for showing current data.